code/game/objects/items.dm

```diff
@@ -317,6 +318,8 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 	for(var/X in actions)
 		var/datum/action/A = X
 		A.Remove(user)
+	if(DROPDEL & flags)

```

Comment:
Why DROPDEL & flags instead of flags & DROPDEL?

Reply:
Just went with TG had; no explicit reason.

Reply:
Could'ya please do it the other way around? It's pretty much consistent in the code that the variable comes before the define when using stuff like == or &.

Reply:
actually its in /tg/'s standards that the variable comes before the consistent too,, this code was out of spec.

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#control-statements

>* All control statements comparing a variable to a number should use the formula of `thing` `operator` `number`, not the reverse (eg: `if (count <= 10)` not `if (10 >= count)`)